packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -272,8 +272,9 @@
         <div class="col-md-4" data-ng-if="$ctrl.technicalDetails">
             <dedicated-server-dashboard-technical-details
                 technical-details="$ctrl.technicalDetails"
-                go-to-manual-upgrade="$ctrl.goToManualUpgrade"
+                go-to-upgrade="$ctrl.goToUpgrade"
```

Comment:
isn't this a method binding?

Reply:
No... We are just passing the method as is